src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeForm/ProPlanControls/BillingControls/BillingControls.spec.tsx

```diff
@@ -0,0 +1,286 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
```

Comment:
This file should be similar to the existing BillingControls test, but this one doesn't consider sentry scenarios